Source/Core/Common/GekkoDisassembler.cpp

```diff
@@ -118,8 +118,6 @@ static const char* regnames[32] = {"r0",  "sp",  "rtoc", "r3",  "r4",  "r5",  "r
 // Initialize static class variables.
 u32* GekkoDisassembler::m_instr = nullptr;
 u32* GekkoDisassembler::m_iaddr = nullptr;
-std::string GekkoDisassembler::m_opcode = "";
```

Comment:
These should still be specified, but just without the initializers, otherwise this can cause compilation errors in the event either std::string instance is taken by reference in the future. Ideally if we we're using C++17 already, it would be fine to mark the relevant member variables as `static inline`, but we aren't yet.

Reply:
done